src/ngx_http_lua_timer.c

```diff
@@ -568,6 +575,9 @@ ngx_http_lua_timer_handler(ngx_event_t *ev)
 
     c = ngx_http_lua_create_fake_connection(tctx.pool);
     if (c == NULL) {
+        ngx_log_error(NGX_LOG_ALERT, ngx_cycle->log, 0,
+                      "failed to create fake connection to run timer co:%p",
```

Comment:
Better use `(co: %p)` (note the space after colon and the surrounding parentheses) here?